neuromation/cli/job.py

```diff
@@ -59,6 +59,9 @@
 
 log = logging.getLogger(__name__)
 
+ENV_VAR_STORAGE_ROOT_MOUNTPOINT = "NP_STORAGE_ROOT_MOUNTPOINT"
```

Comment:
Why do you need the env var at all?

Reply:
we need an env variable here to communicate to a user that the storage tree would be available under a certain directory and be inferred programmatically.

Reply:
We use `/storage/home` constant for `HOME` processing, nobody complains.
Let's use the same strategy.
I prefer to avoid configuration options if they are not requested.

Also, the only already used/provided env var is `NEUROMATION_CONFIG` so far.
`NP_STORAGE_ROOT_MOUNTPOINT` doesn't look consistent.
I know we use `NP_` prefix on *server-side* but people see *client* API/CLI only

Reply:
it is not an env var for `neuro`. it is an env var for jobs.
so, we are not exposing another configuration option, but rather informing users for programmatic usage options.  (same as with `$HOME`).

as the matter of fact, `NP_` was originally used as the platform env var prefix for platform-specific env vars inside jobs. let's rename to whatever is considered consistent.

Reply:
Thank you for correction.
I'm curious, is there a case when a mounting point is not `/var/storage`?
As a user, I prefer to use string constants over env vars if these vars never changes.

Don't get me wrong, I can live with this envvar. The question is: do we need to teach our clients about it?

Reply:
it seems in both cases we will teach users either to hardcode `/var/storage` or hardcode `NEUROMATION_STORAGE_ROOT`. the latter is advantageous to us as we may easily change the value.

Reply:
Ok, let's add a variable that is ignored by most users :)
`NEUROMATION_STORAGE_ROOT` looks good to me.
In this PR, `HOME` is exposed as `${NEUROMATION_STORAGE_ROOT}/home`.
Should we introduce another env var or hardcoded `home` dirname is ok?

Reply:
Personally, I like the idea of exposing `NEUROMATION_STORAGE_ROOT=/var/storage` and re-using it for both `${NEUROMATION_STORAGE_ROOT}/home` and all other mounts.

Reply:
Please note that the method `upload_and_map_config` creates directory `/var/storage/nmrc`, so if a user has directory `storage://~/nmrc`, they will be in conflict. Let's maybe change `local_nmrc_folder` to `/var/storage/.nmrc`?

Reply:
Sounds good